Aakar font seems buggy in the editor

OS: 
Ubuntu, Jammy Jellyfish

Java:
java version "21.0.1" 2023-10-17 LTS
Java(TM) SE Runtime Environment (build 21.0.1+12-LTS-29)
Java HotSpot(TM) 64-Bit Server VM (build 21.0.1+12-LTS-29, mixed mode, sharing)

Alloy Analyser:
6.1.0 built 2021-11-03T15:25:43.736Z

When using the aakar font (which was set as default when I first executed the jar on my computer) the letter "d" will have vertical stripes that stretch across multiple lines. 

This problem is easily solved by selecting another font in "options". Maybe the Analyser could start with another font selected by default?

Text in the editor (when aakar is activated) looks like this:
![image](https://github.com/user-attachments/assets/c1c0c462-448c-4dfc-9733-40574d008f41)

We use a snapshot version of Alloy main branch in the classroom. The students using Alloy with Linux have the following error:

```
Exception in thread "main" java.lang.reflect.InvocationTargetException
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
    at java.base/java.lang.reflect.Method.invoke(Method.java:580)
    at org.alloytools.alloy.core.infra.Alloy.main(Alloy.java:85)
Caused by: java.lang.ExceptionInInitializerError
    at edu.mit.csail.sdg.alloy4.OurSyntaxDocument.<init>(OurSyntaxDocument.java:79)
    at edu.mit.csail.sdg.alloy4.OurSyntaxUndoableDocument.<init>(OurSyntaxUndoableDocument.java:75)
    at edu.mit.csail.sdg.alloy4.OurSyntaxWidget.<init>(OurSyntaxWidget.java:89)
    at edu.mit.csail.sdg.alloy4.OurTabbedSyntaxWidget.newtab(OurTabbedSyntaxWidget.java:460)
    at edu.mit.csail.sdg.alloy4.OurTabbedSyntaxWidget.<init>(OurTabbedSyntaxWidget.java:187)
    at edu.mit.csail.sdg.alloy4whole.SimpleGUI.finishInit(SimpleGUI.java:2154)
    at edu.mit.csail.sdg.alloy4whole.SimpleGUI.<init>(SimpleGUI.java:2066)
    at edu.mit.csail.sdg.alloy4whole.CLIFacade._gui(CLIFacade.java:18)
    at aQute.lib.getopt.CommandLine.execute(CommandLine.java:158)
    at org.alloytools.alloy.core.infra.AlloyDispatcher.__alloy(AlloyDispatcher.java:154)
    at aQute.lib.getopt.CommandLine.execute(CommandLine.java:158)
    at org.alloytools.alloy.core.infra.AlloyDispatcher.main(AlloyDispatcher.java:110)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
    ... 2 more
Caused by: java.util.NoSuchElementException: No value present
    at java.base/java.util.Optional.get(Optional.java:143)
    at org.alloytools.graphics.util.AlloyGraphics.matchBestFontName(AlloyGraphics.java:85)
    at edu.mit.csail.sdg.alloy4.OurConsole.style(OurConsole.java:141)
    at edu.mit.csail.sdg.alloy4.OurConsole.<clinit>(OurConsole.java:84)
    ... 15 more
```

--- a/org.alloytools.alloy.core/src/main/java/org/alloytools/graphics/util/AlloyGraphics.java
+++ b/org.alloytools.alloy.core/src/main/java/org/alloytools/graphics/util/AlloyGraphics.java
@@ -81,7 +81,7 @@ public class AlloyGraphics {
         for (String name : names) {
             String soundex = toSoundex(name);
             Optional<FontFamilyDescription> fontName = availableFontNames.stream().filter(ffd -> ffd.soundex().equals(soundex)).findFirst();
-            if (fontName != null)
+            if (fontName.isPresent())
                 return fontName.get().name;
         }
         return Font.MONOSPACED;

